todo-app/src/components/InputTask.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useRef } from 'react';
+import TodoList from './TodoList';
+import { Task } from '../Type';
+
+const InputTask = () => {
+  const [newTask, setNewTask] = useState('');
+
+  const [todos, setTodos] = useState<Task[]>([]);
+
+  const addTask = (newTask: string) => {
+    if (newTask === '') return;
```

Comment:
この条件式は、以下のように書き換えられます。
```suggestion
    if (!newTask) return;
```

空文字はfalsyな値なためです。
https://developer.mozilla.org/ja/docs/Glossary/Falsy